core/storage/skill/gae_models.py

```diff
@@ -60,6 +60,10 @@ class SkillModel(base_models.VersionedModel):
     skill_contents = ndb.JsonProperty(indexed=False)
     # The id to be used by the next misconception added.
     next_misconception_id = ndb.IntegerProperty(required=True, indexed=False)
+    # The id of the merged skill, in case the skill has been deduped/merged.
```

Comment:
"merged skill" is ambiguous. Perhaps: "The ID of the skill that this one has been merged into (if this skill has been marked as a duplicate of another one)."